SPECS/rubygem-systemd-journal/rubygem-systemd-journal.spec

```diff
@@ -0,0 +1,37 @@
+%global debug_package %{nil}
+%global gemdir %(IFS=: R=($(gem env gempath)); echo ${R[${#R[@]}-1]})
+%global gem_name systemd-journal
+Summary:        Ruby bindings for reading/writing to the systemd journal
+Name:           rubygem-systemd-journal
+Version:        1.3.3
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages
+URL:            https://rubygems.org/gems/%{gem_name}/versions/%{version}
+Source0:        https://rubygems.org/downloads/%{gem_name}-%{version}.gem
+BuildRequires:  ruby >= 1.9.3
+Requires:       rubygem-ffi
+
+%description
+Provides the ability to navigate and read entries from the systemd
+journal in ruby, as well as write events to the journal.
+
+%prep
+%setup -q -c -T
+
+%build
+
+%install
+gem install -V --local --force --install-dir %{buildroot}/%{gemdir} %{SOURCE0}
+
+%files
+%defattr(-,root,root,-)
+%license %{gemdir}/gems/%{gem_name}-%{version}/LICENSE.txt
+%{gemdir}
+
+%changelog
+* Wed Jan 06 2021 Henry Li <lihl@microsoft.com> - 1.3.3-1
+- License verified
+- Original version for CBL-Mariner
```

Comment:
I see that nearly all of these spec files are mentioned here as original, but the complexity above (e.g. ruby >=1.9.3 and the various macros, gives the impression that these were copied from somewhere else....  Are these SPEC files truly original?  If not, we will need to repair the changelogs and be clear on where these originated from.

Reply:
Resolved